package.json

```diff
@@ -85,27 +85,29 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^14.0.0",
     "@trivago/prettier-plugin-sort-imports": "^4.1.1",
-    "@types/node": "^18.15.11",
+    "@types/node": "^18.16.3",
     "@types/p5": "^1.6.1",
-    "@types/react": "^18.0.37",
-    "@types/react-dom": "^18.0.11",
-    "@typescript-eslint/eslint-plugin": "^5.59.0",
+    "@types/react": "^18.2.0",
+    "@types/react-dom": "^18.2.1",
+    "@typescript-eslint/eslint-plugin": "^5.59.2",
     "@vitejs/plugin-react": "^4.0.0",
     "@vitest/coverage-c8": "^0.30.1",
     "canvas": "^2.11.2",
-    "eslint": "^8.38.0",
+    "eslint": "^8.39.0",
     "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
     "gh-pages": "^5.0.0",
     "jsdom": "^22.0.0",
-    "prettier": "^2.8.7",
+    "prettier": "^2.8.8",
     "rimraf": "^5.0.0",
     "typescript": "^5.0.4",
-    "vite": "^4.2.2",
+    "vite": "^4.3.4",
     "vitest": "^0.30.1"
   },
   "dependencies": {
     "microdiff": "^1.3.2",
-    "p5": "^1.6.0"
+    "p5": "^1.6.0",
+    "react": "^18.2.0",
```

Comment:
Do we need to have React as a dependency in the package?

Reply:
I think `PNPM` added these itself for some reason during the `upgrade` command execution cycle or perhaps it was `ncu -u` which did it. Not sure what happened honestly. Either way, something automated, somewhere, made the change but I removed them from dependencies now.

Reply:
Good catch 🚀

Reply:
Need to resolve conflicts when I get onto my laptop, made the change via the mobile app and forgot to update the lock file which is why the CI is on blocked status 😅